modules/gapi/src/backends/oak/goakbackend.cpp

```diff
@@ -375,33 +664,69 @@ cv::gimpl::GOAKExecutable::GOAKExecutable(const ade::Graph& g,
                 m_oak_nodes.at(nh).inputs.reserve(op.args.size());
                 m_oak_nodes.at(nh).outputs.reserve(op.outs.size());
 
+                // Handle Copy operations
```

Comment:
Maybe it's better to prepare `m_oak_nodes` list earlier and don't include copy nodes there so we can freely iterate without these copy exceptions

Reply:
There is a problem with that - we can't get `m_oak_nodes` without running the actual kernels while those depend on copy identification - cyclic dependency. Still, I'll try to think about that

Reply:
Reworked